iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -4,7 +4,7 @@
     <!--See #1874 for additional details-->
     <TargetFrameworks>net5.0;netstandard2.1;netstandard2.0;net472;netcoreapp2.1</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">net5.0;netstandard2.1;netstandard2.0</TargetFrameworks>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Why 8.0?

Reply:
This may have been a merge conflict issue. I'll revert back to 9